bancor_research/bancor_simulator/v3/spec/utils.py

```diff
@@ -429,6 +430,15 @@ def init_protocol(
                 cooldown_time=cooldown_time,
             )
 
+    for tkn_name in ["bnt", "bnbnt", "vbnt"]:
+
+        state.tokens[tkn_name] = Tokens(
+            tkn_name=tkn_name,
+            network_fee=network_fee,
+            withdrawal_fee=withdrawal_fee,
+            cooldown_time=cooldown_time,
+        )
+
     for usr in usernames:
```

Comment:
@mikewcasale - IMO, this is another indication that we should set these configuration parameters for all pools instead of per pool.

Reply:
IMO the argument is synonymous to "we should sell our Lamborghini bc we can't legally drive it any faster than the speed limit"... sometimes the R&D folks just want to "see what happens if we... set things differently by pool". idk, not sure it's a huge issue at the moment but Ill give it some more thought

Reply:
Agreed